finalPrep/annotations.js

```diff
@@ -14,29 +19,41 @@ function buildABear(name, age, fur, clothes, specialPower) {
     sayings: [greeting, powerSaying, "Goodnight my friend!"],
     isCuddly: true,
   };
-
+  //return builtBear should log the key-value pairs defined from lines 15 to line 20
   return builtBear
 }
-
+//pass arguments to the buildABear function that map to parameters : name, age, fur, clothes, and specialPower
 buildABear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares');
+////pass second set of arguments to the buildABear function that map to parameters : 'name', 'age', fur, clothes, and specialPower
 buildABear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in');
 
 
 
 //FizzBuzz
+//this is a function called fizzBuzz with the parameters num1, num2, range
 function fizzBuzz(num1, num2, range) {
+  //create a loop that iterates from 0 to range, incrementing the variable i by 1 each iteration
   for (var i = 0; i <= range; i++) {
+  //if the variable i divided by the parameter num1 is strictly equal to 0 AND the variable i divided by parameter num2 is strictly equal to 0 then:
```

Comment:
Nice explanations of a loop and a modulo. I think annotating code helps focus on areas where some goole research will be beneficial.